sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -628,6 +628,7 @@ describe("LocalWorkspace", () => {
         const refRes = await stack.refresh({ userAgent, previewOnly: true });
         assert.strictEqual(refRes.summary.kind, "update");
         assert.strictEqual(refRes.summary.result, "succeeded");
+        assert.deepStrictEqual(refRes.summary.resourceChanges, { same: 1 });
```

Comment:
Previously, this reported `{ create: 1 }` because the last thing we did was `up`.